components/Init/src/main.c

```diff
@@ -225,8 +225,9 @@ typedef struct memory_range {
 
 static memory_range_t guest_ram_regions[] = {
     /* Allocate all the standard low memory areas */
-    {0x500, 0x7c00 - 0x500},
-    {0x7e00, 0x80000 - 0x7e00},
+    /* The MBR is loaded at 0x7c00, but for this VMM, we don't use MBR
```

Comment:
Even is this example targets x86 anyway, I'd make this a bit more verbose: 
```suggestion
    /* On x86 the BIOS loads the MBR to 0x7c00. But for this VMM, we don't use
     * MBR, so there is no need to exclude the MBR bootstrap code region. */